### DISCUSSION TITLE
CAN SOMEONE MAKE OR HELP ME MAKE A PRESET OR RECOMMEND ME ONE THAT LOOKS SIMILAR PLEASE

### DISCUSSION THREAD

**Discussion by HIKOMORI98** [ntsc-rs/ntsc-rs#137] [Q&A] [answered]
https://github.com/user-attachments/assets/9ff99058-e6f4-4191-805b-5a749ae55cf1

**Comment by Degamisu** [comment]
Ah yes, this looks like a handycam or some sort of Hi8 camcorder. Those are pretty simple to create, but that video is mostly the camera itself doing the artifacts. However, if you combine the composite noise and chroma noise, as well as lowering the bandwidth, it should give you a good outcome.

**Reply by HIKOMORI98** [reply]
so should i use after effects to create that type of effect or ?

**Reply by notdigidani** [reply, marked as answer]
i have a handycam preset for ntsc-rs, here's the link: https://github.com/valadaptive/ntsc-rs/discussions/132 let me know if it helped

**Reply by Degamisu** [reply]
> so should i use after effects to create that type of effect or ?

Which effect? If you mean the noise, just use the standalone application. If you mean the camera, it's pretty hard to replicate that without actually having the camera.
